frontend/app/routes/_index.tsx

```diff
@@ -26,13 +73,92 @@ export const meta: MetaFunction = () => {
     { tagName: 'link', rel: 'canonical', href: pageUrl }
   ]
 }
+
 export default function Index() {
   return (
-    <div className="h-full p-14">
-      <div className="flex flex-wrap justify-center gap-6 max-w-prosex p-4 mx-auto">
-        {availableTools.map((tool) => (
-          <TypeCard key={tool.link} {...tool} />
-        ))}
+    <div className="bg-interface-bg-main min-h-screen flex flex-col gap-2xl items-center pt-5xl pb-xl px-0">
+      <div className="flex flex-col gap-2xl items-center justify-start px-md py-0 w-full max-w-[1280px]">
+        <div className="flex flex-col gap-xs items-center justify-start p-0 w-full">
+          <Typography
+            variant="h1"
+            className="!leading-normal !text-[42px] max-xl:!text-style-h2-semibold text-center"
+          >
+            Web Monetization Tools
+          </Typography>
+          <img
+            alt="Decorative underline"
+            className="block max-w-none h-[23px] w-[442px] max-xl:h-[13px] max-xl:w-[247px]"
+            src={SVGHeadingVector}
+          />
+        </div>
+
+        <Typography
+          variant="h5"
+          className="!text-landing-content text-center w-full max-xl:!text-style-body-standard"
+        >
+          We care about your public and your option you want to help them
+          support you. Feel free to choose any interaction.
+        </Typography>
+      </div>
+
+      <div
+        id="content"
+        className="flex flex-col gap-2.5 items-center justify-start p-0 w-full"
+      >
+        <div className="max-w-[1280px] w-full flex flex-col gap-md items-center justify-start px-md py-0">
+          <Typography variant="h3" className="w-full max-xl:text-center">
+            Setup tools
+          </Typography>
+
+          <Typography
+            variant="h5"
+            className="!text-landing-content text-left w-full max-xl:!text-style-small-standard max-xl:text-center"
+          >
+            We care about your public and your option you want to help them
+            support you. Feel free to choose any interaction.
+          </Typography>
+
+          <div className="flex flex-row items-center justify-between p-0 w-full max-xl:flex-col max-xl:gap-lg max-xl:items-center">
+            {setupTools.map((tool, index) => (
+              <ToolCard
+                key={index}
+                title={tool.title}
+                tags={tool.tags}
+                icon={tool.icon}
+              >
+                {tool.description}
+              </ToolCard>
+            ))}
+            <div className="w-[340px] opacity-0 max-xl:hidden" />
+          </div>
```

Comment:
Between 700-1300px wide viewports, we're showing only one tool in a row, while we can show 2-3 in one row. Intentional?